src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -137,6 +137,11 @@ export default (accessToken, userID, opts) => {
       { method: "PUT" }
     ),
     updateMeLoader: gravityLoader("me", {}, { method: "PUT" }),
+    orderLoader: gravityLoader(
+      ({ id }) => `order/${id}`,
```

Comment:
You can probably just pass the `id` in directly, vs an object that is `{id: ..}` #superminor this is obviously fine.